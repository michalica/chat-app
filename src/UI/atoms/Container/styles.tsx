const styles = {
  root: {
    maxWidth: 640,
    margin: "auto",
  },
};

export default styles;
