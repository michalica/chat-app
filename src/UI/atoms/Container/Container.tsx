import React, { ReactNode } from "react";
import { makeStyles } from "@material-ui/core";
import styles from "./styles";

const useStyles = makeStyles(styles);

const Container = (props: ContainerProps) => {
  const classes = useStyles();

  const { children } = props;
  return <div className={classes.root}>{children}</div>;
};

export default Container;

export interface ContainerProps {
  children: ReactNode;
}
