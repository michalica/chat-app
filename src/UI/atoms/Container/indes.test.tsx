import {shallow} from "enzyme";
import Container from "./Container";

describe("snapshot testing", () => {
    it('should render Container with children correctly', () => {
        expect(shallow(<Container>
            <div>Testing content</div>
        </Container>)).toMatchSnapshot();
    })
})
