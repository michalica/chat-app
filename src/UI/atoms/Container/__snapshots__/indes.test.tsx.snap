// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing should render Container with children correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div>
    Testing content
  </div>
</div>
`;
